packages/ui/src/Button/index.tsx

```diff
@@ -158,7 +158,15 @@ interface RegularProps {
 
 export type ButtonProps = BaseButtonProps & (IconOnlyProps | RegularProps);
 
-/** A component for all your button needs! */
+/**
+ * A component for all your button needs!
+ *
+ * See individual props for how to use `<Button />` in various forms.
+ *
+ * Note that, to use `<Button />` as a link, you can simple wrap it in an `<a
+ * />` (or `<Link />`, if you're using e.g., React Router) tag and leave
+ * `onClick` undefined.
+ */
```

Comment:
I wrapped a button in a link in this PR, so I figured it was worth mentioning that that's possible

Reply:
issue (blocking): console errors for invalid html markup

<img width="913" alt="image" src="https://github.com/user-attachments/assets/f1c111f0-92e6-4981-a9fb-ead9b70e38d2">

Reply:
Oh, weird — the JSDoc syntax I wrote got messed up, causing this error. Fixed [here](https://github.com/penumbra-zone/web/pull/1607/commits/207028b67b7a11becf169fa057775a72b54e1676).